spec/models/tenant_spec.rb

```diff
@@ -457,6 +457,18 @@
       end
     end
 
+    context "sequencing" do
+      it '#sequenceable' do
```

Comment:
The method name is `sequenceable_domains`, so the it should be the same.  Also, you don't need the context around it since you aren't setting anything up and it serves no purpose.